static/sass/patterns/_sections.scss

```diff
@@ -68,6 +67,7 @@
   @include extend('extend-full');
   margin-top: size('spacer');
   text-align: center;
+  z-index: 5;
```

Comment:
we don't have a z-index 3 or 4 anywhere that I could find ... not a big deal, but should we think about including the z-index helper in this project? #someday